lib/screens/movie_home_page.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import '/custom_widgets/movie_body.dart';
+import '/custom_widgets/movie_app_bar.dart';
+import '/data_classes/movie.dart';
+
+class MovieHomePage extends StatelessWidget {
+  MovieHomePage({super.key});
+
+  final Movie movie = Movie.defaultMovie();
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
```

Comment:
You have `MaterialApp` here and on main.dart as well. Remove this and move the properties to the other call, as that's the actual entry point of your app

Reply:
fixed